### ISSUE TITLE
PulseAudio: Mumble causes other audio applications to speed up or slow down

### ISSUE THREAD

**Issue by mkrautz** [mumble-voip/mumble#2162]
bakteria reports on IRC that running Mumble causes other audio-using applications to speed up or slow down.

bakteria suggests that his might be because Mumble is changing the system-wide sampling rate (driver bug? Pulse bug?)

IRC conversation below:

```
20:09:08 <bakteria> Hi there. I have a very strange problem with mumble-git running on arch. Every time I start mumble. It changes my soundcards sample rate. (I belive) I suspect this because all audio eather gets slowed down or fast forwarded.
20:39:17 <bakteria> Just tried the old version as well and its the same.
20:39:32 <bakteria> If I shut down mumble and restart my sound card it works again.
20:55:14 <@mkrautz> bakteria: what audio backend are you using in Mumble?
20:56:55 <bakteria> Using pulseadio and a RME UCX in CC-mode.
20:57:12 <bakteria> Well alsa is probobly the back end
20:57:32 <bakteria> Im confused about pulseaudio
20:58:28 <bakteria> It only happens with mumble. I've used the setup for months and have had no trouble.
20:58:50 <@mkrautz> bakteria: is it ALSA, or is it Pulse?
20:59:13 <bakteria> I've selected pulse in mumble.
20:59:48 <@mkrautz> Mumble always tries to use a 48KHz sample rate, but it'll do resampling if necessary... (you should be able to see the sample rate used by looking at stdout output)
21:00:26 <@mkrautz> if it's Pulse, then I have a hard time seeing how Mumble could do it... how would Mumble be able to set the *system-wide* sampling rate via a PulseAudio session?
21:01:32 <bakteria> I have no idea. I'm not even sure thats whats going on. I have very little experience with audio under linux. It has just worked in the past so I dont fiddle with it.
21:02:04 <@mkrautz> bakteria: try to run Mumble from a terminal so you can look at its output :) I'd like to see a pastebin/gist of it for your audio device. :)
21:02:33 <bakteria> Ok I will do that next time. I've been fiddling with it now for too long and Im sick of it.
21:02:39 <@mkrautz> hehe, alright
21:03:18 <@mkrautz> anyway, I'd really like to help you try to get to the bottom of it, so ping me next time you feel up for it.
21:03:36 <bakteria> Much appreciated! Will do :)
```

**Comment by bakteria1** [comment]
It happened again. 
Started when I opened mumble.
I restarted mumble in verbose mode and this is the output. 
This does not happen every time. In fact its the first time in weeks.

`CELT bitstream 8000000b from /usr/lib/mumble/libcelt0.so.0.7.0
CELT bitstream 80000010 from /usr/lib/mumble/libcelt0.so.0.11.0
Theme: "Mumble"
Style: "Dark"
--> qss: ":builtin/themes/Mumble/Dark.qss"
Locale is "en_US" (System: "en_US")
Database SQLite: "3.11.1"
Overlay: Listening on "/run/user/1000/MumbleOverlayPipe"
Updating application palette
GlobalShortcutX: Using XI2 2.3
AudioInput: 72000 bits/s, 48000 hz, 480 sample
PulseAudio: Starting output: alsa_output.usb-RME_Fireface_UCX__23664304__67E92394A48D3C8-00.analog-stereo
PulseAudio: Starting input alsa_input.usb-RME_Fireface_UCX__23664304__67E92394A48D3C8-00.analog-stereo
AudioOutput: Initialized 1 channel 44100 hz mixer
AudioInput: Initialized mixer for 1 channel 48000 hz mic and 0 channel 48000 hz echo
warning: The VAD has been replaced by a hack pending a complete rewrite`
